coffee-backend/src/test/groovy/chat/letscoffee/util/CookieUtilsSpec.groovy

```diff
@@ -0,0 +1,106 @@
+package chat.letscoffee.util
+
+import org.springframework.mock.web.MockHttpServletRequest
+import org.springframework.mock.web.MockHttpServletResponse
+import spock.lang.Specification
+
+import javax.servlet.http.Cookie
+
+class CookieUtilsSpec extends Specification {
+
+
+    def "getting a cookie from a request should return the correct cookie"() {
+        given: "a request with two cookies"
+        def cookie1 = Mock(Cookie) {
```

Comment:
We create a bunch of Cookies in these tests. Can we create a @Before, or helper function that creates the cookies for us, with 0-2 cookies that can have the same name and attributes every time. That way the tests can focus less on setting up the environment but rather what it is supposed to test.

Reply:
Yupp, will do 👍